week-1/Homework/mandatory/5-magic-8-ball.js

```diff
@@ -45,18 +45,83 @@ Very doubtful.
 
 // This should log "The ball has shaken!"
 // and return the answer.
-function shakeBall() {}
-
-// The answer should come from shaking the ball
-let answer;
 
+const myFuture = [
+  [
+    "It is certain",
+    "It is decidedly so.",
+    "Without a doubt.",
+    "Yes - definitely.",
+    "You may rely on it"
+  ],
+  [
+    "As I see it, yes.",
+    "Most likely.",
+    "Outlook good",
+    "Yes.",
+    "Signs point to yes."
+  ],
+  [
+    "Reply hazy, try again.",
+    "Ask again later.",
+    "Better not tell you now.",
+    "Cannot predict now.",
+    "Concentrate and ask again."
+  ],
+  [
+    "Don't count on it.",
+    "My reply is no.",
+    "My sources say no.",
+    "Outlook not so good.",
+    "Very doubtful."
+  ]
+];
+let a = Math.floor(Math.random() * Math.floor(4));
+let b = Math.floor(Math.random() * Math.floor(5));
+console.log(a);
+function shakeBall() {
+  console.log("The ball has shaken!");
+
+  // The answer should come from shaking the ball
+
+  let answer = myFuture[a][b];
+  return answer;
+  console.log(answer);
+}
 // When checking the answer, we should tell someone if the answer is
 // - very positive
 // - positive
 // - negative
 // - very negative
-function checkAnswer() {}
-
+function checkAnswer() {
+  
+//switch (a) {
+  //case 0:
+    //return  "Very positive";
+    //break;
+  //case 1:
+    //return "Positive";
+    //break;
+  //case 2:
+    //return "Negative";
+    //break;
+  //case 3:
+    //return "Very negative";
+    //break;
+//}
+//}
+if( myFuture[0].includes(answer)){
```

Comment:
You should consider indenting your code to make it more readable. Have you installed Prettier on VS Code yet?